Changes to specs (#177)
yuzu-emu/yuzu-emu.github.io: site/content/help/quickstart/index.md

|   	|  CPU 	|  GPU	|  RAM	|
|-------|-------|-------|-------|

> |**Minimum**|Intel Core i5-4570 / AMD Ryzen 3 1200|Intel HD Graphics 520 / NVIDIA GeForce GT 1030 / AMD Radeon R7 240|8GB|

Reviewer: ```suggestion
|**Minimum**|Intel Core i5-4430 / AMD Ryzen 3 1200|Intel HD Graphics 520 / NVIDIA GeForce GT 1030 / AMD Radeon R7 240|8GB|
```

> |**Recommended**|Intel Core i5-10400 / AMD Ryzen 5 3600|NVIDIA GeForce GTX 1060 6GB / AMD Radeon RX 470 8GB|16GB| 

Reviewer: ```suggestion
|**Recommended**|Intel Core i7-8700 / AMD Ryzen 5 3600|NVIDIA GeForce GTX 1060 6GB / AMD Radeon RX 470 8GB|16GB| 
```

It would be better if we did not suggest this particular CPU as the recommended spec

Author: the 8700 is more expensive for the same result, basically, Intel just now has a decent 6 core lineup.

- Our recommended specifications don't guarantee perfect performance in most games, but rather strive to provide a cost effective recommendation while still considering performance.

> - Mobile CPUs will not reach the same performance as desktop parts.

Reviewer: ```suggestion
- Mobile CPUs will not reach the same performance as their desktop counterparts due to thermal, power, and technical limitations. 
```
